game/scripts/vscripts/components/points/points.lua

```diff
@@ -17,5 +24,81 @@ function PointsManager:Init ()
   end
 
   -- set initial values for current scores
-  CustomNetTables:SetTableValue( "team_scores", "score", { radiant = 0, dire = 0 } )
+  CustomNetTables:SetTableValue( "team_scores", "score", { goodguys = 0,
+                                                           badguys = 0
+                                                         })
+
+  -- Start Thinking
+  Timers:CreateTimer(0, Dynamic_Wrap(PointsManager, "Think"))
+end
+
+function PointsManager:Think ()
+  DebugPrint("[points/PointsManager] Thinking..")
```

Comment:
don't print on think, makes it so that you need to disable this whole module when working on another one